docs/handbook/getting-started.md

```diff
@@ -1532,6 +1530,133 @@ Use the following steps to change the root password:
 * Once changed you can exit the su session
 * You should be able to login/authenticate as root now.
 
+## Updating OpenIndiana /dev to /hipster
+
+This section describes how to update your existing OpenIndiana /dev installation to OpenIndiana /hipster.
+
+Note, that although direct updating from /dev to /hipster can be possible, it's not exhaustively tested.
+
+There are several techniques which you can use to update your systems, including root pool installation from OpenIndiana ISO.
+Complete reinstall also can be a decent option.
+
+Further it's considered that you were warned and decided to do more-or-less direct update from /dev to /hipster.
+
+<i class="fa fa-info-circle fa-lg" aria-hidden="true"></i> **NOTE:**
+<div class="well">
+Only server installation update was tested.
+If you do GUI installation update, you are on your own.
+Of course, you are welcome to ask questions in <a href="https://openindiana.org/mailman/listinfo/oi-dev">oi-dev</a> mailing list, but prepare that nobody will guide you through update.
+</div>
+
+* Do backup.
+  <br/>Seriously.
+  The next steps can make your system unbootable.
+
+* Read release notes.
+  <br/>Read release notes for ALL OpenIndiana Hipster snapshots.
+  They contain information on system changes and possible troubles which can appear.
+  Read [Loader Integration](https://www.openindiana.org/2016/09/28/loader-integration/)
+
+* Test your actions in non-production environment.
+  <br/>Consider that last significant OpenIndiana /dev update was many years ago.
+  You are going to update your applications perhaps via several major releases.
+  Prepare for the changes.
+
+* You'll need console access to the server.
+  <br/>Without console access and ability to boot fresh OpenIndiana Hipster ISO image, you'll not be able to activate new boot environment: old tools can't work with new facilities, new tools will not work with old kernel.
+
+* Update to the latest OpenIndiana /dev version.
+
+```
+# pkg update -v
+```
+
+* Create new boot environment, which you'll update to /hipster, and mount it.
+
+```bash
+# beadm create oi-hipster
+# beadm mount oi-hipster /mnt
```

Comment:
The `/mnt` point can be used for hosting "other-device" mounts, at least is on systems I've seen, like temporary network shares, LOFSes of CD/DVD images, etc.

I think to be consistent with Solarish practice, the common `/a` is a better choice for alternate BE mount.

Reply:
According to `filesystem(5)`:
```
/mnt
          Default temporary mount point for file systems. This is an empty
           directory on which file systems can be temporarily mounted.
```

Reply:
Huh, ok. If so says the standard... maybe word a warning a bit above, like "...and mount it into /mnt (the default empty temporary mountpoint directory)" so if a reader has it set up or used otherwise, they should adapt accordingly.